packages/codegen-ui/lib/__tests__/generate-form-definition/helpers/datastore-model.test.ts

```diff
@@ -68,6 +80,7 @@ describe('addDataStoreModelField', () => {
       isArray: false,
     };
 
-    expect(() => addDataStoreModelField(formDefinition, {}, dataStoreModelField)).toThrow();
+    // adding any so no TypeError is thrown as opposed to a ValidationError
```

Comment:
I think you can check for a specific error message instead to avoid using any.

Reply:
I want to keep this as a many so we can get the actual error coming through in the event something happens at runtime we want the errors to work as expected.